_tests/integration/version_test.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	expectedCLIVersion = "1.49.1"
+	expectedCLIVersion = "1.50.0"
```

Comment:
I feel like this is a bug fix, so next version should: `1.49.1`

Reply:
👍 Current version is `1.49.2`, so I'll update it to `1.49.3`.

Reply:
Updated in https://github.com/bitrise-io/bitrise/pull/796/commits/3cc2296ac2dc04f822c553553a55f01898cd5b71